File: CoreJava/interface-crud-app/src/components/DeptDB.java
package components;

public class DeptDB implements ICrudabe {

	@Override
	public void create() {
		System.out.println("DeptDB created.");

	}

	@Override
	public void read() {
		System.out.println("DeptDB Read.");

	}

	@Override
	public void update() {
		System.out.println("DeptDB update.");

	}

	@Override
	public void delete() {
		System.out.println("DeptDB Deleted.");

	}

}

File: C#/OOP/InterfaceCRUD-App/InterfaceCRUD-App/DeptDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InterfaceCRUD_App
{
    class DeptDB : Icrudable
    {
      
	public void create() {
        Console.WriteLine("DeptDB created.");

	}


	public void read() {
        Console.WriteLine("DeptDB Read.");

	}


	public void update() {
        Console.WriteLine("DeptDB update.");

	}

	public void delete() {
        Console.WriteLine("DeptDB Deleted.");

	}
    }
}
